planner/core/testdata/plan_suite_out.json

```diff
@@ -2081,19 +2081,38 @@
     "Cases": [
       {
         "SQL": "select /*+nth_plan(1)*/ * from test.tt where a=1 and b=1",
-        "Plan": "TableReader(Table(tt)->Sel([eq(test.tt.a, 1) eq(test.tt.b, 1)]))"
+        "Plan": [
+          "TableReader_7 0.01 root  data:Selection_6",
+          "└─Selection_6 0.01 cop[tikv]  eq(test.tt.a, 1), eq(test.tt.b, 1)",
+          "  └─TableFullScan_5 10000.00 cop[tikv] table:tt keep order:false, stats:pseudo"
+        ]
       },
       {
         "SQL": "select /*+nth_plan(2)*/ * from test.tt where a=1 and b=1;",
-        "Plan": "IndexLookUp(Index(tt.a)[[1,1]], Table(tt)->Sel([eq(test.tt.b, 1)]))"
+        "Plan": [
+          "IndexLookUp_11 0.01 root  ",
+          "├─IndexRangeScan_8(Build) 10.00 cop[tikv] table:tt, index:a(a) range:[1,1], keep order:false, stats:pseudo",
+          "└─Selection_10(Probe) 0.01 cop[tikv]  eq(test.tt.b, 1)",
+          "  └─TableRowIDScan_9 10.00 cop[tikv] table:tt keep order:false, stats:pseudo"
+        ]
       },
       {
         "SQL": "select /*+nth_plan(3)*/ * from test.tt where a=1 and b=1;",
-        "Plan": "IndexLookUp(Index(tt.b)[[1,1]], Table(tt)->Sel([eq(test.tt.a, 1)]))"
+        "Plan": [
+          "IndexLookUp_15 0.01 root  ",
+          "├─IndexRangeScan_12(Build) 10.00 cop[tikv] table:tt, index:b(b) range:[1,1], keep order:false, stats:pseudo",
+          "└─Selection_14(Probe) 0.01 cop[tikv]  eq(test.tt.a, 1)",
+          "  └─TableRowIDScan_13 10.00 cop[tikv] table:tt keep order:false, stats:pseudo"
+        ]
       },
       {
         "SQL": "select /*+nth_plan(2)*/ * from test.tt where a=1 and b=1;",
-        "Plan": "IndexLookUp(Index(tt.a)[[1,1]], Table(tt)->Sel([eq(test.tt.b, 1)]))"
+        "Plan": [
+          "IndexLookUp_11 0.01 root  ",
+          "├─IndexRangeScan_8(Build) 10.00 cop[tikv] table:tt, index:a(a) range:[1,1], keep order:false, stats:pseudo",
+          "└─Selection_10(Probe) 0.01 cop[tikv]  eq(test.tt.b, 1)",
+          "  └─TableRowIDScan_9 10.00 cop[tikv] table:tt keep order:false, stats:pseudo"
+        ]
```

Comment:
Please add a case to test that the queries with and without `nth_plan()` hint result the same execution plan.

Reply:
I added a test case for a query without `nth_plan()` hint which has the same output, but I'm not sure if it's appropriate because I can't make sure these two outputs wouldn't be modified unintentionally to be different.

Reply:
how about storying the execution plan result in a string and comparing these two strings?

Reply:
OK. Added a test case for that.